File: app/src/main/java/models/UserNetwork.java
package models;

import java.util.ArrayList;

public final class UserNetwork {
    private static ArrayList<String> followers;
    private static ArrayList<String> following;
    private static ArrayList<String> subscribed;

    public static ArrayList<String> getFollowers() {
        return followers;
    }

    public static void setFollowers(ArrayList<String> mFollowers) {
        followers = mFollowers;
    }

    public static ArrayList<String> getFollowing() {
        return following;
    }

    public static void setFollowing(ArrayList<String> mFollowing) {
        following = mFollowing;
    }

    public static ArrayList<String> getSubscribed() {
        return subscribed;
    }

    public static void setSubscribed(ArrayList<String> mSubscribed) {
        subscribed = mSubscribed;
    }
}

File: app/src/main/java/com/sqube/tipshub/models/UserNetwork.kt
package com.sqube.tipshub.models

import kotlin.collections.ArrayList

object UserNetwork {
    var followersList: ArrayList<String>? = null
    var followingList: ArrayList<String>? = null
    var subscribedList: ArrayList<String>? = null

    val followers get() = followersList ?: arrayListOf()
    val following get() = followingList ?: arrayListOf()
    val subscribed get() = subscribedList ?: arrayListOf()
}